docs/WizardExample.jsx

```diff
@@ -11,13 +11,20 @@ class AddressStep extends React.Component {
   render() {
     const {setWizardState, wizardState, recipient} = this.props;
     return (
-      <form ref={(f) => { this.form = f; }} onSubmit={(e) => e.preventDefault()}>
+      <form
+        ref={f => {
+          this.form = f;
+        }}
+        onSubmit={e => e.preventDefault()}
+      >
```

Comment:
Ah it was triggering since I set a max width of 80 chars on my own editor, I think it blindly applied it to everythign